llvm/cmake/config-ix.cmake

```diff
@@ -257,6 +257,37 @@ else()
   set(LLVM_ENABLE_TERMINFO 0)
 endif()
 
+#Check for icu.
+if(LLVM_ENABLE_ICU)
+  set(LIBRARY_SUFFIXES ${CMAKE_FIND_LIBRARY_SUFFIXES})
+  set(CMAKE_FIND_LIBRARY_SUFFIXES ".so")
+  if (LLVM_ENABLE_ICU STREQUAL FORCE_ON)
+    find_package(ICU REQUIRED COMPONENTS uc i18n)
+    if (NOT ICU_FOUND)
+      message(FATAL_ERROR "Failed to configure icu, but LLVM_ENABLE_ICU is FORCE_ON")
+    endif()
+  else()
+    find_package(ICU COMPONENTS uc i18n)
+  endif()
+  set(HAVE_ICU ${ICU_FOUND})
+  set(CMAKE_FIND_LIBRARY_SUFFIXES ${LIBRARY_SUFFIXES})
+endif()
+
+# Check for iconv.
+if(LLVM_ENABLE_ICONV)
```

Comment:
Are we missing CMake changes to link against iconv?

Reply:
I think since we are now checking for Iconv_IS_BUILT_IN, we do not need the link step anymore. Please let me know if you think we still need the link step. Thanks!

Reply:
Ah, that's entirely possible this just works (I'm familiar with cmake but by no means an expert). Have you locally tested that you get no link errors when enabling iconv and it's already installed on the system? (and I sort of wonder whether this means we should have a both set up to build with ICU and with iconv to ensure all tests pass with either library)

Reply:
Yes, I tried this out on a system with iconv installed and it worked without the link step.